src/CSharp/EasyMicroservice.FileManager/Extensions/FileExtensions.cs

```diff
@@ -2,7 +2,7 @@
 using System.IO;
 using System.Threading.Tasks;
 
-namespace EasyMicroservice.FileManager
+namespace EasyMicroservice.FileManager.Extensions
```

Comment:
For the 'Extensions' We don't use the 'extensions' namespace because of developer usage will be harder.